cmd/evm/internal/t8ntool/flags.go

```diff
@@ -30,10 +30,13 @@ var (
 		Name:  "trace",
 		Usage: "Output full trace logs to files <txhash>.jsonl",
 	}
-	TraceDisableMemoryFlag = &cli.BoolFlag{
-		Name:  "trace.nomemory",
-		Value: true,
-		Usage: "Disable full memory dump in traces (deprecated)",
+	TraceTracerFlag = &cli.StringFlag{
+		Name:  "trace.tracer",
+		Usage: "The type of tracer. It could be callTracer or prestateTracer",
+	}
+	TraceTracerConfigFlag = &cli.StringFlag{
+		Name:  "trace.tracer.config",
```

Comment:
maybe we can just name this flag as `trace.jsonconfig`. The namespace `trace` means this flag is used to specify tracer, `jsonconfig` means the value contains the config in json format?

Reply:
sounds good to me